Cakefile

```diff
@@ -35,6 +35,11 @@ build = (callback) ->
             console.error err
         else
             build_webpack () ->
+                jsPath = build_dir + '/APIBlueprintImporter.js'
+                js = fs.readFileSync(jsPath, 'utf-8')
+                js = js.replace('var Module={"ready"', 'var Module={"ENVIRONMENT":"WEB","ready"')
```

Comment:
Update drafter.js to not override Modules in https://github.com/apiaryio/drafter.js/releases/tag/v2.6.4. However I still can't prepend `Module` ENVIRONMENT with webpack. The [`DefinePlugin`](https://webpack.github.io/docs/list-of-plugins.html#defineplugin) doesn't appear to work.

I've updated to drafter.js 2.6.4 in https://github.com/apiaryio/Paw-APIBlueprintImporter/pull/20/commits/61e1d01eed8b3cba74909b99cdbae4ccc31ae114 which means changing this hack slightly.

Not sure what else I can do, perhaps we can leave this hack as-is for now and continue? We are version pinned and the Paw extension won't if `ENVIRONMENT` isn't set to `WEB`.